src/traccuracy/metrics/_base.py

```diff
@@ -46,6 +48,19 @@ def compute(self, matched: Matched) -> Results:
         Returns:
             Results: Object containing metric results and associated pipeline metadata
         """
+        if override_matcher:
```

Comment:
Is the intent of this to allow skipping the validation? I think it just prints the warning and doesn't actually skip it

Reply:
You are entirely correct and code review is a wonderful thing :joy: